sdk/loadtestservice/load-testing-rest/samples/v1-beta/typescript/src/sample.ts

```diff
@@ -109,7 +110,10 @@ async function main() {
   if (testRunCreationResult.body.testRunId === undefined)
     throw new Error("Test Run ID returned as undefined.");
 
-  const testRunPoller = await getTestRunCompletionPoller(client, testRunCreationResult);
+  const testRunPoller = await getLongRunningPoller(client, testRunCreationResult);
```

Comment:
testRunCreationResult can be defaultResponse as well? how does the long running poller handle this?

Reply:
We are checking the error response before this on line #107, for unexpected response, LRO is called only when there is successful response.